Cubical/Categories/Constructions/Relators.agda

```diff
@@ -0,0 +1,55 @@
+{-# OPTIONS --safe #-}
+{-
+
+  Category whose objects are bifunctors and morphisms are natural
+  transformations.
+
+-}
+module Cubical.Categories.Constructions.Relators where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Functor.Base
+open import Cubical.Categories.NaturalTransformation
+open import Cubical.Categories.Bifunctor.Redundant
+open import Cubical.Categories.Instances.Functors
+open import Cubical.Categories.Instances.Sets
+open import Cubical.Categories.Constructions.ChangeOfObjects
+open import Cubical.Categories.Constructions.Bifunctors
+open import Cubical.Categories.Profunctor.Relator
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' ℓE ℓE' : Level
+
+open NatTrans
+
+module _ (C : Category ℓC ℓC') (D : Category ℓD ℓD') ℓR where
+  RELATOR : Category _ _
+  RELATOR = BIFUNCTOR (C ^op) D (SET ℓR)
```

Comment:
Maybe I'll add the bicategory of profunctors since we seem to have a decent amount of definitions in place